core/pom.xml

```diff
@@ -19,18 +19,23 @@
             <name>bintray</name>
             <url>https://jcenter.bintray.com</url>
         </repository>
+        <repository>
+            <id>dv8tion</id>
+            <name>m2-dv8tion</name>
+            <url>https://m2.dv8tion.net/releases</url>
+        </repository>
```

Comment:
Since you are adding a new maven repository here, can we remove one of the old ones.
(Assuming this is for discord4j)

Reply:
pretty sure its only needed for lavaplayer dunno if anything is depending on the old repo so i dint remove it yet

cleaning stuff like that up is why i opened issue #247